src/app/components/right-menu/quickstart/quickstart.component.html

```diff
@@ -6,6 +6,8 @@
             allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
     <p>If you have suggestions, comments, requests, or if you find any bugs, please let us know by email: <br/>
       <em>b.von-stengel [at] lse.ac.uk</em> <br/> <em>antonov.martin [at] gmail.com</em></p>
+    <p>The symbolic sequential eqilibrium solver was developed by Moritz Graf.<br/>
```

Comment:
This should be moved to the log of the algorithm (as done with the backwards induction algorithm )